scipy/stats/biasedurn.pxd

```diff
@@ -0,0 +1,20 @@
+from numpy.random cimport bitgen_t
+
+# Declare the class with cdef
+cdef extern from "biasedurn/stocc.h" nogil:
+    cdef cppclass CFishersNCHypergeometric:
+        CFishersNCHypergeometric() except +
+        CFishersNCHypergeometric(int, int, int, double, double) except +
+        int mode()
+        double mean()
+        double variance()
+        double probability(int x)
+        double moments(double * mean, double * var)
+
+    cdef cppclass StochasticLib3:
+        StochasticLib3() except +
+        StochasticLib3(int seed) except +
+        void SetBitGen(bitgen_t *that_bitgen_state)
```

Comment:
This must be called before the class is used to ensure that the correct bitgen_t object is being used